### ISSUE TITLE
code: creating the first classifier

### ISSUE THREAD

**Issue by clay-arras** [UBCAgroBot/target-object-localization#6]
## Overview

Create a simple classifier. It doesn’t need to be complicated — just enough to identify basic classes. Since we’re keeping things simple, make it a binary classifier.

The datasets we use will most likely be multi-class, so we'll need to convert the labels to what binary classifiers expect. Choose one class to be the positive instance (X). All other classes will be treated as negative examples. This should output the confidence that the model has that the image is X. 

We won’t be able to test this classifier yet with real images, because even after #5 we’ll still need to resize them (classifiers expect fixed-size images). However, as a dry run, here are two things you can do to somewhat test the classifier is set up correctly:

1. Run a random tensor through the model:
   ```
   torch.randn(B, C, X, Y)
   ```
   and check if it gives an output and doesn't crash (for now lets assume batch_size=32, channels=3, x=64, y=64).

2. Another dry run: run that random tensor through the model, train it with `.backward()` and cross-entropy loss, and see if you can get a training loss of 0. If not, the model is probably too simple.

**Comment by ericlyfan** [comment]
implemented in 9ab40e3
